udata/forms/fields.py

```diff
@@ -540,8 +540,9 @@ def _value(self):
             return ''
 
     def process_formdata(self, valuelist):
-        if not valuelist:
```

Comment:
you may keep this condition. Else a case with valuelist = None won't match the new first condition but then `len(None)` will fail